src/Routing/AuthenticatedContentRouteSubscriber.php

```diff
@@ -53,6 +53,23 @@ protected function alterRoutes(RouteCollection $collection) {
         $route->setRequirement('_custom_access', 'tide_authenticated_content.access_jsonapi_checker::access');
       }
     }
+    $route_config = \Drupal::config('tide_authenticated_content.route_settings');
+    $route_settings = $route_config->get("route_settings");
```

Comment:
```suggestion
    $route_settings = $route_config->get('route_settings');
```
avoid using double-quotes for literal strings.